### PR TITLE
Fix wrong octicon displayed for closed PR

### LINKED ISSUE #75: Closed PR shown as Merged
**Describe the bug**

image
The PR AY2223S1-CS2103T-T09-4/tp/pull/376 is closed instead of being merged. But in the view of WATcher as below (highlighted in red box), the PR has the icon as being merged instead of being closed.

Screenshot 2023-02-06 at 1 25 12 PM

**To Reproduce**

1. Go to WATcher
2. Open "AY2223S1-CS2103T-T09-4/tp" repository
3. See error

**Expected behavior**
A closed PR should have its icon same as GitHub closed PR and differ from merged PR.

**Desktop (please complete the following information):**

- OS: MacOS
- Browser Chrome
- Version 109

### PR DESCRIPTION
**Pull request by Echomo-Xinyu**
### Summary:

Fixes #75 

Let's update the octicon and its colour for the closed PR.

### Changes Made:

- Fix octicon and colour for closed PR

Before:

image

After:

Screenshot 2023-03-05 at 1 50 43 AM

As highlighted in the red box, the octicon icon and corresponding card view color has been updated to fit the GitHub convention.

### Commit Message:

```
Let's update the octicon and its colour for the closed PR
```

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -150,5 +150,8 @@
     "@schematics/angular:directive": {
       "prefix": "app"
     }
+  },
+  "cli": {
+    "analytics": false
```
Comment (cheehongw) [inline angular.json:155]: is this change related to the PR?

Comment (Echomo-Xinyu) [reply angular.json:155]: No, they are not related to this PR and are side products during the local test. Thanks for pointing it out!

File: src/app/issues-viewer/card-view/card-view.component.ts [hunk 2]
```diff
@@ -109,7 +121,11 @@ export class CardViewComponent implements OnInit, AfterViewInit, OnDestroy {
       case type === 'PullRequest' && state === 'OPEN': {
         return 'git-pull-request';
       }
-      case type === 'PullRequest': {
+      case type === 'PullRequest' && state === 'CLOSED': {
+        return 'git-pull-request-closed';
+      }
+      case type === 'PullRequest' && state === 'MERGED': {
+        // case type === 'PullRequest': {
```
Comment (kkangs0226) [inline src/app/issues-viewer/card-view/card-view.component.ts:128]: Let's remove this commented line

**Review by kkangs0226 — COMMENTED** [review]
Good work, seems to be working well! 👍 

Side note, for UI changes, remember to add before & after images in the description under "Changes made".

**Review by kkangs0226 — APPROVED** [review]
LGTM!
